pkg/api/breakdownResolver.go

```diff
@@ -0,0 +1,26 @@
+package api
+
+import (
+	"github.com/thrasher-redhat/internal-tools/pkg/db"
+)
+
+// Breakdown of a given bug query
+// Contains Total bugs, new bugs, and closed bugs
+
+// TODO - Change naming scheme to better represent what things are?
```

Comment:
Rollup contains 3 breakdowns for:

* All bugs
* Blocker Bugs
* Bugs with customer cases

Breakdowns each contain:

* All bugs
* New bugs
* Closed bugs


The all/total bugs can be confusing since it exists in both rollups and breakdowns.  Figure out two good names (descriptive and distinct) and use them.

Reply:
Rollups contain ALL

Breadowns contain TOTAL